### ISSUE TITLE
Windows False positive?

### ISSUE THREAD

**Issue by Kiapo** [andro2157/DiscordTokenProtector#40]
Getting a false positive possibly from windows defender? unsure. 

Program:Win32/Uwamson.A!ml

Sorry not techy :(

**Comment by andro2157** [comment]
Hello,

That's definitely a false positive, it happens each time I update DTP.
I'll report that issue to the Windows Defender team.

**Comment by GlitchedCod** [comment]
![image](https://user-images.githubusercontent.com/44437316/167305282-7e25eba0-7d14-4696-a286-aec47c6cf98a.png)
There is a new false positive ! (Sorry I'm in French Windows 10 Pro)

**Comment by fritzhaxx** [comment]
I'm sure @andro2157 can read French well 😬

**Comment by andro2157** [comment]
> ![image](https://user-images.githubusercontent.com/44437316/167305282-7e25eba0-7d14-4696-a286-aec47c6cf98a.png) There is a new false positive ! (Sorry I'm in French Windows 10 Pro)

Yeah, that's the new signature after I reported it several times to Microsoft.
They don't seem to like the fact that DTP is reading the memory of Discord to get the user's token in the initial setup.

**Comment by dekrypted** [comment]
I think some of these may be the cause:
- DLL Hooking
- Protected Processes
- Elevation
- Memory Reading

However, they're false.
